.github/scripts/wait-for-checks.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+#
+# @param $1 - PR number or URL
+# wait for a bit until pr is created, otherwise it throws an error "no checks reported on the 'odh-release/e2e-test' branch"
+set -euo pipefail
+
+sleep 10
+
+while [ $(gh pr checks "$1" | grep -v 'tide' | grep 'pending' | wc -m) -gt 0 ]; do
+  printf "PR checks still pending, retrying in 10 seconds...\n"
+  sleep 30 # replace with 60000
+done
+
+if [ $(gh pr checks "$1" | grep 'fail' | wc -m) -gt 0 ]; then
```

Comment:
ditto.

But it looks that you are interested in the second column only, you can grep some other parts by accident.
Would the wrapper work? 
```
# would be shorter if negated, just exit 1 in if
# choose better name
status_exists() {
    gh pr checks $1 | awk "{if (\$2 == \"$2\") found=1} END{if (!found) exit 1}"
}

# probably you do not expect PR number with spaces?
status_exists $1 fail && { echo "check failed"; exit 1}
```